static/docs/user-guide/install/dev.md

```diff
@@ -0,0 +1,45 @@
+# Install for testing and development
+
```

Comment:
This is also duplicated in https://dvc-org-pr-695.herokuapp.com/doc/user-guide/contributing#development-environment, but with totally different methods...

Reply:
Maybe I should remove the installation for development and testing, to avoid duplication: https://dvc-org-pr-695.herokuapp.com/doc/user-guide/install/dev

Reply:
Yes, thanks.